spec/helpers/user_helpers.rb

```diff
@@ -0,0 +1,34 @@
+module UserHelpers
+
+  def add_user(user)
+    User.create(email: user.email,
+                password: user.password,
+                password_confirmation: user.password_confirmation,
+                real_name: user.real_name,
+                user_name: user.user_name)
+  end
+
+  def sign_up(user)
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for sign_up is too high. [17/15]